polar-core/src/polar.lalrpop

```diff
@@ -99,15 +99,21 @@ Variable: Value  = <n:Name> => {
     Value::Variable(n)
 };
 
-Predicate: Value = {
+Call: Value = {
     <name:Name> "(" <mut args:(<TermExp> ",")*> <arg:TermExp?> ")" => {
```

Comment:
off-topic: does this technically mean that `f(1,)` is valid Polar?

Reply:
```
query> f(1,)
False
```
yerp

Reply:
Fixed